api-docs/time-series-introduction.md

```diff
@@ -49,20 +49,24 @@ Explore the available endpoints and interact with the Time Series API to retriev
+   Their names consist of the `machine_insight_` prefix followed by lowercase insight name separated with underscores (`_`). See [Data Model](data-model) for a list of all possible machine insights.
 2. **Standard CAN inputs and outputs** - for example `advanced_sensor_input_1` or `advanced_sensor_run_1`.  
-   Their names start with `advanced_sensor_` prefix followed by input number (1-10) or run number (1-6).
-3. **Other sensors** originating from various sources capturing metrics that are uniquely defined for specific equipment. For example: `advanced_sensor_work_light_switch_front_left_status`.  
-   If applicable, the unit of measurement can be read from `unit_of_measurement` label.  
-   Some of the sensors might generate multiple series with minimum, maximum and average values. In such case the series name will contain suffix `_min`, `_max` or `_avg`.
+   Their names start with the `advanced_sensor_` prefix followed by an input number (1-10) or a run number (1-6).
+3. **Other sensors** can originate from various sources and capture metrics that are uniquely defined for specific equipment. For example: `advanced_sensor_work_light_switch_front_left_status`.  
+   If applicable, the unit of measurement can be read from the `unit_of_measurement` label.  
+   Some of these sensors might generate multiple series with minimum, maximum and average values. In such cases the series name will contain the suffix `_min`, `_max` or `_avg`.
 
 When querying advanced sensor data, you can specify a unique sensor ID to retrieve metrics for a particular sensor mapping. For example: `{sensor_id="run_1"}` or `{sensor_id="variable_11231"}`.
 
-Every asset has its own combination of available series and labels depending on type of the equipment and its configuration. It is not possible to list all the names in this documentation, however the API provides 3 endpoints to list [time series names](getlistoftimeseries), [label names](getlistoflabelnames) and [label values](getlistoflabelvalues) for specific asset.
+Every asset has its own combination of available series and labels depending on the type of the equipment and its configuration. It is not possible to list all the names in this documentation, however the API provides 3 endpoints to list [time series names](getlistoftimeseries), [label names](getlistoflabelnames) and [label values](getlistoflabelvalues) for a specific asset.
+
+> 📘 Prometheus & special characters
+>
+> Please be aware that Prometheus interprets certain characters in other ways than you might expect. For example: if an advanced sensor name contains paranthesis (), you will need to escape and URLencode these paranthesis to create a valid request. This means that `query=advanced_sensor_roading_high_(%)` has to turn into `query=advanced_sensor_roading_high_%5C%28%25%5C%29` instead.
```

Comment:
1) Typo in "paranthesis"

2) URL-encoding ([percent-encoding](https://en.wikipedia.org/wiki/Percent-encoding)? I don't think I've ever heard about percent-encoding before googling) is only needed when the query is in the URL, so it's only needed for `GET`.

`POST` requests have query in the body, so there's no need for URL-encoding, but there's still need to escape characters with `\`

3) Thorsten wrote to me on Slack with more questions from Manitou and it looks like `%` must be escaped as well.

So the query should be:
* GET: `advanced_sensor_handling_high_%5C%28%5C%25%5C%29`
* POST: `advanced_sensor_handling_high_\(\%\)`